frontend/src/Explorer/Util/String.js

```diff
@@ -9,3 +9,9 @@ exports.substituteImpl = function(str) {
     });
   };
 };
+
+exports.take = function (n) {
```

Comment:
@akegalj Kristijan and me had a chat about it and `bower i purescript-strings#2.1.0 --save` will work.